src/aldente_server.cpp

```diff
@@ -1,10 +1,68 @@
 #include "aldente_server.h"
 
+#include "util/config.h"
+#include "game/game_state.h"
+#include "physics.h"
+#include "scene_manager.h"
+#include "net/network_manager.h"
 #include "timer.h"
 #include <chrono>
 #include <iostream>
+#include <boost/filesystem.hpp>
+
+
+AldenteServer::~AldenteServer() {
+    GeometryGenerator::destroy();
+    glfwTerminate();
+}
+
+static void glSetup() {
```

Comment:
Sure.  Kavin says he'll work on this once #54 is merged in to avoid conflicts in `grid.cpp`.  I'll leave this PR on standby in the meantime.